### PR TITLE
Vulcan Anticheat hook

### PR DESCRIPTION
**Pull request by OmeWillem**
I've added a hook for the Vulcan anticheat, should work just fine. Uses the API jar that they provide (doesn't seem to have a public repo anywhere)

### CODE REVIEW

File: veinminer-bukkit/src/main/java/wtf/choco/veinminer/anticheat/AntiCheatHookVulcan.java
Comment (2008Choco) [inline veinminer-bukkit/src/main/java/wtf/choco/veinminer/anticheat/AntiCheatHookVulcan.java:35]: Proper braces around if statements please
```java
if (...) {
    return;
}
```

File: veinminer-bukkit/src/main/java/wtf/choco/veinminer/anticheat/AntiCheatHookVulcan.java
Comment (2008Choco) [inline veinminer-bukkit/src/main/java/wtf/choco/veinminer/anticheat/AntiCheatHookVulcan.java:33]: For consistency, I'd rather this not be final. I don't tend to mark parameters as final

File: veinminer-bukkit/src/main/java/wtf/choco/veinminer/VeinMinerPlugin.java
Comment (2008Choco) [inline veinminer-bukkit/src/main/java/wtf/choco/veinminer/VeinMinerPlugin.java:29]: All imports should be kept, please :)

**Review by 2008Choco — CHANGES_REQUESTED** [review]
I really don't like using local files as dependencies. I do have a repository on which I host some other anti cheat APIs (such as Spartan and Themis), so I'd honestly rather do that. Obviously you don't have push access to my local repository, but I can update the dependency after the merge. I have no issue with that. Alternatively, I think I'm able to edit this pull request directly? Or I can make a suggestion and have you change it once I get the artifact uploaded.

I'll revisit this PR this weekend and get things published.

File: veinminer-bukkit/src/main/java/wtf/choco/veinminer/VeinMinerPlugin.java
Comment (OmeWillem) [reply veinminer-bukkit/src/main/java/wtf/choco/veinminer/VeinMinerPlugin.java:29]: I understand the other changes, but just for curiosity. Why are you against using .*?

**Review by 2008Choco — APPROVED** [review]
Everything looks solid to me now I think. Like I said in a previous comment, I'll revisit some time this weekend and publish the artifact.

**Comment by 2008Choco** [comment]
Thanks a lot :) I'm not sure when this will get rolled out, but I will be sure to credit when it does.

**Pull request merged**
